LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Francesco Tabaro
```

Comment:
Please update the copyright to: (c) 2020 NykterLab (at: https://ruoho.uta.fi/wp/)